conf/uno.conf

```diff
@@ -16,9 +16,9 @@ export FLUO_YARN_VERSION=1.0.0
 # you must also change the hash below.
 export FLUO_HASH=037f89cd2bfdaf76a1368256c52de46d6b9a85c9c1bfc776ec4447d02c813fb2
 export FLUO_YARN_HASH=c6220d35cf23127272f3b5638c44586504dc17a46f5beecdfee5027b5ff874b0
-export HADOOP_HASH=$(grep $HADOOP_VERSION $UNO_HOME/conf/hadoop/sha | cut -d ' ' -f 2)
-export ZOOKEEPER_HASH=$(grep $ZOOKEEPER_VERSION $UNO_HOME/conf/zookeeper/sha | cut -d ' ' -f 2)
-export ACCUMULO_HASH=$(grep $ACCUMULO_VERSION $UNO_HOME/conf/accumulo/sha | cut -d ' ' -f 2)
+export HADOOP_HASH=$(grep -F hadoop:${HADOOP_VERSION} $UNO_HOME/conf/sha | cut -d : -f 3)
```

Comment:
These could be more specific, as in `hadoop:${HADOOP_VERSION}:` (note the trailing colon)

Reply:
Fixed in bffdf6f8b2